### PR TITLE
updated and enhanced the footer added icon with the link , added copyright label and overall giving a better look. #15

### PR DESCRIPTION
**Pull request by Madan2468**
enhanced the footer added icon with the link , added copyright label and overall giving a better look that matches with the theme.

updation  of footer #15 

#### Brief description of what is fixed or changed
The resumegenerator footer section was modified , added font awesome CDN link at the top of head. Updated style.css.

#### Other comments
This was my first contribution , hope this suited you. Thank you!!

### CODE REVIEW

**Comment by Kishan-Ved** [comment]
Thanks for this, I'll review it within 24 hours.

**Comment by Madan2468** [comment]
sure do lemme know if any changes required.

File: resumegenerator.html [hunk 1]
```diff
@@ -666,32 +671,56 @@ <h5 class="text-center my-2">A rough preview. Differs from the LaTeX output. <br
...
+                        <i class="fas fa-envelope"></i> kishan.ved@iitgn.ac.in
+                    </li>
+                </ul>
+            </div>
+        </div>
+        <div class="footer-follow">
+            <p>Follow Us:</p>
+            <ul class="follow-icons">
+                <li>
+                    <a href="https://github.com/Kishan-Ved" target="_blank">
```
Comment (Kishan-Ved) [inline resumegenerator.html]: Please replace this link by https://github.com/Kishan-Ved/resume_generator

**Comment by Kishan-Ved** [comment]
For some phones, like Samsung S8+, **IIT Gandhinagar** and the gmail link are not aligned one below the other. Please try to fix this, it will make the footer more responsive and professional.

image

**Comment by Kishan-Ved** [comment]
You can make changes and push code on this PR's branch itself, it will automatically be reflected in this PR.

**Comment by Madan2468** [comment]
Sure i will do the changes , kindly add the tag hacktoberfest-accepted tag.

File: resumegenerator.html (hunk 1 above)
Comment (Madan2468) [reply resumegenerator.html]: Sure

**Comment by Kishan-Ved** [comment]
> Sure i will do the changes , kindly add the tag hacktoberfest-accepted tag.

I'll add the hacktoberfest-accepted tag once this PR is merged.

**Comment by Madan2468** [comment]
I have updated , have a look
@Kishan-Ved

**Comment by Kishan-Ved** [comment]
You forgot to change the hyperlink. I have done that. Responsiveness of the footer is good.

**Review by Kishan-Ved — APPROVED** [review]

**Comment by Kishan-Ved** [comment]
Thanks for this PR! I have added the hacktoberfest and hacktoberfest-accepted labels, please check your profile on Hacktoberfest's website and let me know if you can see this PR.

Also, please star this repo, it will increase reach and we will have more contributors :)

**Comment by Madan2468** [comment]
sure ! Thank you @Kishan-Ved looking forward to more Projects.

**Pull request merged**